### PR TITLE
Use showcase v0.0.7

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -109,10 +109,10 @@ anchor_run_ruby: &anchor_run_ruby
   working_directory: /tmp/
   docker:
     - image: circleci/ruby:2.4-node
+    - image: gcr.io/gapic-showcase/gapic-showcase:0.0.6
```
Comment (andreamlin) [inline .circleci/config.yml]: Which commands use this image?

Comment (landrito) [reply .circleci/config.yml]: No commands use this but rather this spins up this image to run in the background. The way that these image configurations work in circleci is a bit confusing. The first listed image is the image that the subsequent steps are run in. The following images are images that are run in the background and are networked together. Essentially this is causing gapic-showcase to run in the background which causes the showcase integration tests to be able to make calls to it.

**Comment by landrito** [comment]
Ah it seems that the ruby test command was lost somewhere in the mix.

File: .circleci/config.yml (hunk 1 above)
Comment (andreamlin) [reply .circleci/config.yml]: oh neato

**Pull request merged**